Source/KerbalCombatSystems/Utils.cs

```diff
@@ -6,56 +6,35 @@ namespace KerbalCombatSystems
 {
     public static partial class KCS
```

Comment:
Please could you undo the function moves for now? Or give me a summary of what you changed? The diff is impossible to read. The original order was:

1 FindDecoupler
2 VesselDistance
3 GetMaxAcceleration
4 GetMaxThrust
5 RayIntersectsVessel
6 TargetLead
7 TryToggle
8 FindDecouplerChildren
9 FromTo
10 RelVel
11 AngularAcceleration
12 Integrate
13 SolveTime
14 SolveDistance
15 AngularVelocity
16 ShortenName
17 FindCommand
18 FindController
19 AveragedSize
20 FuelMass
21 DryMass

Reply:
No changes to the existing functions. Newer order is
#Get Properties
1 AveragedSize
2 Fue Mass
3 DryMass
4 GetMaxAcceleration
5 GetMaxThrust
6 GetFireVector
7 GetMeanVector(Possibly a poorly named function but gets the thrust direction for ksp parts that don't have it correlating to up)
# DoAction
8 ShortenName
9 TryToggle
# Finders
10 FindCommand
11 FindController
12 FindDecoupler
13 FindDecouplerChldren
# Physics Calcs
14 FromTo
15 RelVel
16 AngularAcceleration
17 AngularVelocity
18 Integrate
19 SolveTime
20 SolveDistance
21 TargetLead
22 VesselDistance
23 RayIntersectsVessel

Reply:
Issue was the difficulty in rolling it back successfully

Reply:
I was able to see a proper diff by committing the function moves on my branch and it all looks good.